package com.practice.core.exceptionHandling;

public class Throws5 {

	public static void main(String[] args) {
		try {
			System.out.println("Hello");
		}catch(ArithmeticException e) {
			
		}
	}
}
